cni/pkg/ebpf/main/main.go

```diff
@@ -0,0 +1,45 @@
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"net/http"
+
+	"istio.io/istio/cni/pkg/ebpf/server"
+	"istio.io/pkg/log"
+)
+
+func main() {
```

Comment:
is this deadcode now? and if so, do we need most of the redirect server which is handling HTTP requests?

Reply:
yes, dead code now, we can remove it